lib/index.js

```diff
@@ -416,6 +416,20 @@ Segment.prototype.referrerId = function(query, ctx) {
   this.cookie('s:context.referrer', json.stringify(ad));
 };
 
+/**
+ * isCrossDomainAnalyticsEnabled returns true if cross domain analytics is enabled.
+ * This field is not directly supplied, so it is inferred by inspecting the
+ * `crossDomainIdServers` array in settings. If this array is null or empty,
+ * it is assumed that cross domain analytics is enabled.
```

Comment:
This comment seems wrong. Why is xid enabled when crossDomainIdServers is null or empty? The code below (line 431) seems to suggest the reverse.

Reply:
yep - I updated this! it meant to say disabled.